.github/workflows/deploy.yml

```
@@ -0,0 +1,37 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - main
+jobs:
+  container-job:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Cache purescript dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.npm
+            .spago
+            ${{ env.spago_global_cache }}
+            output
+          key: >-
+            ${{ hashFiles('package-lock.json') }}-
+            ${{ hashFiles('spago.dhall') }}-
+            ${{ hashFiles('packages.dhall') }}
+      - name: Install dependencies
+        run: npm ci --loglevel verbose
+      - name: Build
+        run: npm run build
+      - name: Test
+        run: npm t
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GH_TOKEN }}
```

Per [docs](https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-set-runners-access-token-github_token), if we use `GITHUB_TOKEN` instead, it is provisioned [by GitHub](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) with [read/write](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#enabling-workflows-for-private-repository-forks) permissions. :-)

```suggestion
          github_token: ${{ secrets.GITHUB_TOKEN }}
```

p.p. ams